### PR TITLE
Set diff editor border to match regular editor border. 

### PR DESCRIPTION
**Pull request by itamark**
Set diff editor border to match regular editor border. 
Solves issue with no shadow showing no diff between editors.
Solves [#53523](https://github.com/Microsoft/vscode/issues/53523)

### CODE REVIEW

File: src/vs/platform/theme/common/colorRegistry.ts [hunk 1]
```diff
@@ -280,6 +280,8 @@ export const diffRemoved = registerColor('diffEditor.removedTextBackground', { d
 export const diffInsertedOutline = registerColor('diffEditor.insertedTextBorder', { dark: null, light: null, hc: '#33ff2eff' }, nls.localize('diffEditorInsertedOutline', 'Outline color for the text that got inserted.'));
 export const diffRemovedOutline = registerColor('diffEditor.removedTextBorder', { dark: null, light: null, hc: '#FF008F' }, nls.localize('diffEditorRemovedOutline', 'Outline color for text that got removed.'));
 
+export const diffBorder = registerColor('diffEditor.diffEditorBorder', { dark: '#444444', light: '#E7E7E7', hc: contrastBorder }, nls.localize('diffEditorBorder', 'Border color of separation between original and modified diff editors.'));
```
Comment (sa-adebayo) [inline src/vs/platform/theme/common/colorRegistry.ts]: The description could be better written as 'Border color between the original and modified diff editors.'
What do you think?

**Comment by sa-adebayo** [comment]
You could adjust the PR description as well for those who want to know what was done at first glance.

```
Set diff editor border to match regular editor border. 
Solves issue with no shadow showing no diff between editors.
Solves [#53523](https://github.com/Microsoft/vscode/issues/53523)
```

Does it look clearer, if not please adjust.

File: src/vs/platform/theme/common/colorRegistry.ts (hunk 1 above)
Comment (itamark) [reply src/vs/platform/theme/common/colorRegistry.ts]: That's a much better description! Done.

**Comment by joaomoreno** [comment]
Thanks, renamed it to `diffEditor.border`. :beers:

**Pull request merged**